insights/parsers/numeric_user_group_name.py

```diff
@@ -0,0 +1,62 @@
+"""
+NumericUserGroupName - command ``/usr/bin/grep -c '^[[:digit:]]' /etc/passwd /etc/group``
+=========================================================================================
+
+This parser reads the output of ``/usr/bin/grep -c '^[[:digit:]]' /etc/passwd /etc/group``
```

Comment:
Will fix it.

In my opinion, this is a simple grep, a simple parser (and a simple rule), should be rather fast and lightweight parser & rule.